pkg/reduce/reduce.go

```diff
@@ -35,10 +34,13 @@ import (
 	"github.com/numaproj/numaflow/pkg/udf/applier"
 	"github.com/numaproj/numaflow/pkg/watermark/fetch"
 	"github.com/numaproj/numaflow/pkg/watermark/publish"
+	"github.com/numaproj/numaflow/pkg/window"
 )
 
 // DataForward reads data from isb and forwards them to readloop
 type DataForward struct {
+	vertexName          string
```

Comment:
how about we just keep `*dfv1.Vertex` and derive the rest at the usage site?

Reply:
Done.